src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => {};
+import { FC } from 'react';
+import { IUser } from '../../types/User';
+
+type TUserInfo = {
```

Comment:
T-type notation is old fashioned and don't really give us any benefits like in C#/Java so you can simplify it without T.

```suggestion
type UserInfoProps = {
```

Reply:
Not fixed